registry/v2/registry.go

```diff
@@ -63,49 +74,44 @@ func (r *registry) ManifestExists(ctx echo.Context) error {
 	ref := ctx.Param("reference") // ref can be either tag or digest
 
 	manifest, err := r.store.GetManifestByReference(ctx.Request().Context(), namespace, ref)
-
 	if err != nil {
 		details := echo.Map{
 			"error":   err.Error(),
 			"message": "skynet - manifest not found",
 		}
 
 		errMsg := r.errorResponse(RegistryErrorCodeManifestBlobUnknown, err.Error(), details)
-		echoErr := ctx.JSONBlob(http.StatusNotFound, errMsg)
 		r.logger.Log(ctx, fmt.Errorf("%s", errMsg))
-		return echoErr
+		return ctx.NoContent(http.StatusNotFound)
 	}
 
-	metadata, err := r.skynet.Metadata(manifest.Skylink)
+	metadata, err := r.dfs.Metadata(GetManifestIdentifier(namespace, manifest.Reference))
 	if err != nil {
 		detail := map[string]interface{}{
 			"error":   err.Error(),
-			"skylink": manifest.Skylink,
+			"dfsLink": manifest.DFSLink,
 		}
 
 		errMsg := r.errorResponse(RegistryErrorCodeManifestBlobUnknown, "Manifest does not exist", detail)
-		echoErr := ctx.JSONBlob(http.StatusNotFound, errMsg)
 		r.logger.Log(ctx, fmt.Errorf("%s", errMsg))
-		return echoErr
```

Comment:
aren't we kinda losing the log here, since it won't be recorded in echoErr anymore?